lib/memjs/memjs.js

```diff
@@ -82,12 +82,18 @@ Client.create = function(serversStr, options) {
   return new Client(servers, options);
 };
 
+// An overridable method you can use for determing
+// server selection. Should return the server index
+// in the list of servers on Client#servers.
```

Comment:
Could you add the usage example to the comment?

Reply:
Just added the example, let me know if it needs changing, thanks!